gamebook/helpers.py

```diff
@@ -1,43 +1,53 @@
 import random
 import gamebook.trait
+from gamebook.terminal_output import TerminalOutput
 
 
-def openscrn(screen):
+def open_screen(screen):

```

Comment:
What does this do? why is there a function for this? will this be more complicated in the future?

Reply:
don't think so...it originally had the opening screen string inside it but you told me to move it to `example.py`
i'll erase it :+1: